variables.tf

```diff
@@ -247,3 +247,13 @@ variable "tunnel2_ike_versions" {
   type        = list(string)
   default     = null
 }
+variable "local_ipv4_network_cidr" {
+  description = "(Optional) The IPv4 CIDR on the customer gateway (on-premises) side of the VPN connection."
+  type        = string
+  default     = null
+}
+variable "remote_ipv4_network_cidr" {
+  description = "(Optional) The IPv4 CIDR on the AWS side of the VPN connection."
+  type        = string
+  default     = null
```

Comment:
Same here, I would suggest to use [the default value for the remote IPv4 CIDR provided by AWS](https://docs.aws.amazon.com/vpn/latest/s2svpn/VPNTunnels.html#:~:text=Remote%20IPv4%20Network%20CIDR) here, which is `0.0.0.0/0`